From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 12 Oct 2016 19:05:42 -0400
Subject: make documentation reproducible by omitting HTML_TIMESTAMP

we want the same docs to come out each time we run the build process.
---
 lang/qt/doc/Doxyfile.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lang/qt/doc/Doxyfile.in b/lang/qt/doc/Doxyfile.in
index 5d94f90..577f845 100644
--- a/lang/qt/doc/Doxyfile.in
+++ b/lang/qt/doc/Doxyfile.in
@@ -1147,7 +1147,7 @@ HTML_COLORSTYLE_GAMMA  = 80
 # The default value is: YES.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_TIMESTAMP         = YES
+HTML_TIMESTAMP         = NO
 
 # If the HTML_DYNAMIC_SECTIONS tag is set to YES then the generated HTML
 # documentation will contain sections that can be hidden and shown after the
